### ISSUE TITLE
New rule: originalUriBaseIds must end with a slash

### ISSUE THREAD

**Issue by lgolding** [microsoft/sarif-sdk#1854]
See microsoft/sarif-visualstudio-extension#127. The spec requires the trailing slash. Some consumers might compensate for its absence, but the producer can't rely on that.

@michaelcfanning

**Comment by eddynaka** [comment]
@lgolding , i think this issue is already done.

The rule SARIF1018 has the following validation:
```csharp
if (uriString != null && Uri.IsWellFormedUriString(uriString, UriKind.RelativeOrAbsolute))
{
    var uri = new Uri(uriString, UriKind.RelativeOrAbsolute);

    if (artifactLocation.UriBaseId == null && !uri.IsAbsoluteUri)
    {
        LogResult(pointer, nameof(RuleResources.SARIF1018_NotAbsolute), uriString, uriBaseId);
    }

    if (!uriString.EndsWith("/"))
    {
        LogResult(pointer, nameof(RuleResources.SARIF1018_LacksTrailingSlash), uriString, uriBaseId);
    }
}
```

**Comment by lgolding** [comment]
This was fixed by #1858. Thanks @eddynaka for pointing that out.
